### PR TITLE
Fix dist

### PR DESCRIPTION
**Pull request by minsooshin**

### CODE REVIEW

File: dist/styles/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { default as button } from './buttons.css';
+export { default as form } from './form.css';
+export { default as link } from './links.css';
+export { default as util } from './utils.css';
```
Comment (RaulZhao) [inline dist/styles/index.js:4]: Need an empty line

**Review by RaulZhao — APPROVED** [review]
+1

File: dist/styles/index.js (hunk 1 above)
Comment (minsooshin) [reply dist/styles/index.js:4]: this is generated file by gulp, so I have no control.

**Comment by RaulZhao** [comment]
BTW, `node scripts/start.js` is not working.

**Comment by minsooshin** [comment]
@RaulZhao what do you mean by `node scripts/start.js not working`? this toolset should be run by `yarn run docs`

File: dist/styles/index.js (hunk 1 above)
Comment (minsooshin) [reply dist/styles/index.js:4]: will take a look at a solution with gulp later.

**Pull request merged**